src/main/java/com/google/sps/servlets/MatchServlet.java

```diff
@@ -49,8 +65,24 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //return null for matches so that page redirects to logged out homepage
         response.getWriter().println(gson.toJson(null));
     } else {
-        Collection<User> matches = matchRepository.getMatchesForUser(sessionContext.getLoggedInUser());
-        response.getWriter().println(gson.toJson(matches));
+        // calling analyze request 
+        boolean value = analyzeRequest(request);
+        // if true, means the request was passed
```

Comment:
L70: Here, on the other hand, adding a comment (as you have) makes more sense, but it'd be more helpful if it explained what it means for the request to have passed (giving the reader a concise picture of why a request might or might not pass)

Reply:
Done